package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "grpc-gateway",
+  "version": "1.0.0",
+  "description": "[![release](https://img.shields.io/github/release/grpc-ecosystem/grpc-gateway.svg?style=flat-square)](https://github.com/grpc-ecosystem/grpc-gateway/releases) [![CircleCI](https://img.shields.io/circleci/project/github/grpc-ecosystem/grpc-gateway/master.svg?style=flat-square)](https://circleci.com/gh/grpc-ecosystem/grpc-gateway) [![coverage](https://img.shields.io/codecov/c/github/grpc-ecosystem/grpc-gateway/master.svg?style=flat-square)](https://codecov.io/gh/grpc-ecosystem/grpc-gateway) [![license](https://img.shields.io/github/license/grpc-ecosystem/grpc-gateway.svg?style=flat-square)](LICENSE.txt)",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs",
+    "example": "examples"
+  },
+  "scripts": {
+    "git:commit": "git-cz && ./node_modules/validate-commit-msg/lib/cli.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/grpc-ecosystem/grpc-gateway.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/grpc-ecosystem/grpc-gateway/issues"
+  },
+  "homepage": "https://github.com/grpc-ecosystem/grpc-gateway#readme",
+  "devDependencies": {
+    "cz-conventional-changelog": "^2.1.0",
+    "husky": "^1.3.1",
+    "validate-commit-msg": "^2.14.0"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    },
+    "validate-commit-msg": {
+      "types": [
+        "feat",
+        "fix",
+        "docs",
+        "style",
+        "refactor",
+        "perf",
+        "test",
+        "chore",
+        "revert",
+        "ci",
+        "build"
+      ],
+      "warnOnFail": false,
+      "maxSubjectLength": 100,
+      "subjectPattern": "^[A-Za-z]+-[0-9]+((,| - ).*)?",
+      "subjectPatternErrorMsg": "Wrong commit format! Subject does not match subject pattern!",
+      "helpMessage": ""
+    }
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  }
+}
```

Comment:
Why did we add this? I don't think we want to publish an npm module.

Reply:
The same. It provides me git commit hook so that developers are forced to commit messages in a standard format. See git-cz.

Reply:
Please remove it.

Reply:
copy it